### DISCUSSION TITLE
classifying: Good first issue

### DISCUSSION THREAD

**Discussion by vasnt** [seaweedfs/seaweedfs#5851] [Q&A]
are there any good-first-issue?

https://github.com/seaweedfs/seaweedfs/issues
I can see there are 300+ issues atm.

**Comment by chrislusf** [comment]
Many issues are really old. Maybe help to clean them up could be one good first issue. :)

**Reply by vasnt** [reply]
Sure, can you elaborate (step by step if possible), how to do it? i am quite new, for  both coding & seaweedfs.

If possible, let's prioritise which issues are important to do first, second likewise...
